src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -15,14 +15,23 @@ const StyledRadioButtonContainer = styled.label`
   user-select: none;
   width: fit-content;
   ${props => props.disabled && disabledStyle} ${props =>
-    !props.disabled &&
-    'cursor: pointer;'}
+  !props.disabled && 'cursor: pointer;'}
 
   :hover input:not([disabled]) + div,
   :hover input:not([disabled]) + span {
     border-color: ${props =>
       normalizeColor(props.theme.radioButton.hover.border.color, props.theme)};
   }
+  :not(div):hover {
```

Comment:
Trying to understand this selector a little more. How come we're saying `:not(div)`?